crates/nu-command/src/filters/lines.rs

```diff
@@ -76,27 +110,15 @@ impl Command for Lines {
 
                 Ok(PipelineData::ListStream(stream, metadata))
             }
-            PipelineData::Value(val, ..) => {
-                match val {
-                    // Propagate existing errors
-                    Value::Error { error, .. } => Err(*error),
-                    _ => Err(ShellError::OnlySupportsThisInputType {
-                        exp_input_type: "string or raw data".into(),
-                        wrong_type: val.get_type().to_string(),
-                        dst_span: head,
-                        src_span: val.span(),
-                    }),
+            PipelineData::ByteStream(stream, ..) => {
+                if let Some(lines) = stream.values() {
```

Comment:
Just in case you do decide to change `Values`, keep in mind this needs to change too